resources/styles/components/course-calendar/header.less

```diff
@@ -33,6 +26,25 @@
         border-top-right-radius: 0;
       }
     }
+
+    .-no-periods-text {
+      display: inline-block;
+      margin: 16px 0;
+    }
+
```

Comment:
I kinda thought we weren't supposed to use `-` prefixed classes for styling.  They're for test selection only right, and if we use them for styling we're supposed to remove the `-`